### DISCUSSION TITLE
Vivo In Vacuo - a space exploring project.

### DISCUSSION THREAD

**Discussion by AWachtendorf** [hajimehoshi/ebiten#1510] [Show and tell]
Seems like I'm first to show and tell about something :)

I'm working on a space exploring game that I call Vivo in Vacuo (should be more or less latin for "I'm living in a vacuum").

[Youtube screencast Demo](https://youtu.be/c9ufCQMT1h8)
[Github Repo](https://github.com/AWachtendorf/VivoInVacuo)

Currently all graphics are mockups or just rectangles. 
Atm the game has health and shield bars, floating objects, particle animations, an inventory, collectable items, a minimap , questmarker system and displays in which sector the ship currently resides. 

I'm currently adding more documentation and try to clean the code.

Open for feedback, tipps or contributions :)

**Comment by hajimehoshi** [comment]
Nice! Would it be possible to play this on browsers?

**Reply by AWachtendorf** [reply]
I really was thinking about this, as I realized that all your examples run in browsers.
At this point it's kind of too early to say I'm afraid :-/
Did you design your example page yourself?

**Reply by hajimehoshi** [reply]
OK I'm looking forward to it!

> Did you design your example page yourself?

Yes.
